common/src/main/java/com/wynntils/models/wynnitem/parsing/WynnItemParser.java

```diff
@@ -190,11 +190,11 @@ public static WynnItemParseResult parseItemStack(
                 String starString = statMatcher.group(5);
                 String statDisplayName = statMatcher.group(6);
 
+                // Skill bonuses looks like stats when parsing, ignore them
+                if (Skill.isSkill(statDisplayName)) continue;
```

Comment:
This is a low hanging optimization, which I've done to make the performance impact of the regex-based display names lookups close to 0.